frontend/src/settings/display/ExampleDisplay.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Button } from '@mui/material'
+import { ThemeProvider } from '@mui/material/styles'
+import { patchCurrentUserSettings, useGetCurrentUserSettings } from 'actions/user'
+import { ThemeKey } from 'src/theme'
+
+interface ExampleDisplayProps {
+  theme: ThemeKey
```

Comment:
This prop name is confusing as it's not a `Theme`. Consider renaming to `themeMapping` in line with one of my other comments.